instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Comment from './Comment';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import './comment.css';
+
+const CommentSection = (props) => {
+    return (
+        <section className="comment-container">
+            <section className="comment-icons">
+                <FontAwesomeIcon icon='heart' />
+                <FontAwesomeIcon icon='comment-alt' />
+            </section>
+            <p><strong>{props.likes} likes</strong></p>
+            <section className="comments">
+                {props.comments.map(comment => (
+                    <div>
```

Comment:
do you need the `Comment` component wrapped in a div here? And where's your 🔑 ❓ 🕵️